### ISSUE TITLE
cargo - nakamoto build error

### ISSUE THREAD

**Issue by urza** [coffee-tools/folgore#79]
Hi,

following these instructions from readme:

```
git clone https://github.com/vincenzopalazzo/nakamoto.git && cd nakamoto
git checkout macros/client_model-fixes
cd node
cargo run -- --log debug --testnet
```

results in these errors:
![image](https://github.com/coffee-tools/folgore/assets/189804/871583ee-64a6-46d5-98e2-d0a1aee0c96b)

**Comment by urza** [comment]
After googling the error I found [the solution](https://stackoverflow.com/questions/52445961/how-do-i-fix-the-rust-error-linker-cc-not-found-for-debian-on-windows-10):

`sudo apt install build-essential`

Maybe this hint should be added to the readme. I am on fresh ubuntu 22.04 so it is probably very common that others will run into this as well.

**Comment by vincenzopalazzo** [comment]
>Maybe this hint should be added to the readme. I am on fresh ubuntu 22.04, so it is probably very common that others will also run into this.

Yeah, I think we should specify that currently, we are not able to create a sandbox (e.g with nix) so the host OS should contain all the pre-requisite for compiling the plugin,

Another docs issue for @tareknaser
